app/pb_kits/playbook/pb_date_range_inline/_date_range_inline.jsx

```diff
@@ -25,41 +33,112 @@ const dateTimeIso = (dateValue) => {
 }
 
 const DateRangeInline = (props: DateRangeInlineProps) => {
-  const { endDate, startDate } = props
+  const {
+    icon = false,
+    dark = false,
+    size = 'sm',
+    align = 'left',
+    startDate,
+    endDate,
+  } = props
+
+  const iconContent = () => {
+    return (
+      <If condition={icon}>
+        <Body
+            color="light"
+            tag="span"
+        >
+          <Icon
+              className="pb_date_range_inline_icon"
+              dark={dark}
+              fixedWidth
+              icon="calendar-alt"
+              size={size}
+              tag="span"
+          />
+        </Body>
+      </If>
+    )
+  }
+
+  const dateInCurrentYear = () => {
+    const currentDate = new Date()
+    return startDate.getFullYear() == endDate.getFullYear() && startDate.getFullYear() == currentDate.getFullYear()
+  }
+
+  const renderTime = (date) => {
+    return (
+      <time dateTime={dateTimeIso(date)}>
+        <Choose>
+          <When condition={dateInCurrentYear()}>
+            {` ${dateTimestamp(date, false)} `}
+          </When>
+          <Otherwise>
+            {` ${dateTimestamp(date, true)} `}
+          </Otherwise>
+        </Choose>
+      </time>
+    )
+  }
+
   return (
-    <div className={classnames('pb_date_range_inline', globalProps(props))}>
-      <Body
-          color="light"
-          tag="span"
-      >
-        <Icon
-            fixedWidth
-            icon="calendar-alt"
-        />
-      </Body>
-      <Body tag="span">
-        <time dateTime={dateTimeIso(startDate)}>
-          {` ${dateTimestamp(
-          startDate
-        )} `}
-        </time>
-      </Body>
-      <Body
-          color="light"
-          tag="span"
-      >
-        <Icon
-            fixedWidth
-            icon="long-arrow-right"
-        />
-      </Body>
-      <Body tag="span">
-        <time dateTime={dateTimeIso(endDate)}>
-          {` ${dateTimestamp(
-          endDate
-        )} `}
-        </time>
-      </Body>
+    <div className={classnames('pb_date_range_inline_kit_' + align, globalProps(props))}>
```

Comment:
We have a function that does the concatenation joining.. 
It's called `buildCss` -- so in the past we've combined them. 
I think the way you have it here works. But it might be good to stick with our established patterns. 

```suggestion
    <div className={classnames(dateRangeClasses, globalProps(props)}>
```